posthog/api/element.py

```diff
@@ -35,27 +38,33 @@ class ElementViewSet(viewsets.ModelViewSet):
     ]
     permission_classes = [IsAuthenticated, ProjectMembershipNecessaryPermissions]
 
-    def get_queryset(self) -> QuerySet:
-        queryset = super().get_queryset()
-
-        return queryset.filter(group__team=self.request.user.team)
+    def filter_queryset_by_parents_lookups(self, queryset):
```

Comment:
This is quite complicated - would it perhaps make sense to override get_queryset instead and not call super in there?

Reply:
`get_queryset` wouldn't work, but I put the rewriting logic into `StructuredViewSetMixin`